runtime/compiler/control/J9Options.cpp

```diff
@@ -1566,7 +1566,7 @@ void J9::Options::preProcessMode(J9JavaVM *vm, J9JITConfig *jitConfig)
             {
             UDATA aggressivenessValue = 0;
             IDATA ret = GET_INTEGER_VALUE(argIndex, aggressiveOption, aggressivenessValue);
-            if (ret == OPTION_OK && aggressivenessValue >= 0)
```

Comment:
It looks like at one point this tested for `aggressivenessValue > 0` because 0 indicated that `_aggressivenessLevel` had not been set.  When the unset value was later changed to -1, this comparison changed to check for a non-negative value.

As things stand, however, the fact that an unsigned `aggressivenessValue` is assigned to a signed `_aggressivenessLevel` might lead to unexpected behaviour.

Running this in the debugger:

```
gdb --args ./jdk/bin/java -XaggressivenessLevel4294967293 -version
...
1569                if (ret == OPTION_OK && aggressivenessValue >= 0)
(gdb) p aggressivenessValue
$3 = 4294967293
(gdb) n
1571                   _aggressivenessLevel = aggressivenessValue;
(gdb) n
1576       }
(gdb) p _aggressivenessLevel
$4 = -3
```

I'm wondering whether this needs to guard against overflow.  Marius @mpirvu, any thoughts?